### PR TITLE
Update description to tidb_scatter_region

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
Update description to tidb_scatter_region

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/7466 https://github.com/pingcap/docs-cn/pull/7622
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1446,6 +1446,7 @@ SET tidb_query_log_max_len = 20
 - Scope: GLOBAL
 - Default value: `OFF`
 - By default, Regions are split for a new table when it is being created in TiDB. After this variable is enabled, the newly split Regions are scattered immediately during the execution of the `CREATE TABLE` statement. This applies to the scenario where data need to be written in batches right after the tables are created in batches, because the newly split Regions can be scattered in TiKV beforehand and do not have to wait to be scheduled by PD. To ensure the continuous stability of writing data in batches, the `CREATE TABLE` statement returns success only after the Regions are successfully scattered. This makes the statement's execution time multiple times longer than that when you disable this variable.
+- Note that if `SHARD_ROW_ID_BITS` or `PRE_SPLIT_REGIONS` has been set when a table is created, the specified number of Regions are evenly split after the table creation.
```
Comment (shichun-0415) [inline system-variables.md:1449]: 根据研发老师提供的参考，`SHARD_ROW_ID_BITS` 和 `PRE_SPLIT_REGIONS` 是要同时配置才能生效的，参考：https://docs.pingcap.com/zh/tidb/v5.0/sql-statement-split-region#pre_split_regions

Comment (TomShawn) [inline system-variables.md:1449]:
```suggestion
- Note that if `SHARD_ROW_ID_BITS` and `PRE_SPLIT_REGIONS` has been set when a table is created, the specified number of Regions are evenly split after the table creation.
```

File: system-variables.md [hunk 2]
```diff
@@ -1446,6 +1446,7 @@ SET tidb_query_log_max_len = 20
 - Scope: GLOBAL
 - Default value: `OFF`
 - By default, Regions are split for a new table when it is being created in TiDB. After this variable is enabled, the newly split Regions are scattered immediately during the execution of the `CREATE TABLE` statement. This applies to the scenario where data need to be written in batches right after the tables are created in batches, because the newly split Regions can be scattered in TiKV beforehand and do not have to wait to be scheduled by PD. To ensure the continuous stability of writing data in batches, the `CREATE TABLE` statement returns success only after the Regions are successfully scattered. This makes the statement's execution time multiple times longer than that when you disable this variable.
+- Note that if `SHARD_ROW_ID_BITS` and `PRE_SPLIT_REGIONS` has been set when a table is created, the specified number of Regions are evenly split after the table creation.
```
Comment (TomShawn) [inline system-variables.md:1449]:
```suggestion
- Note that if `SHARD_ROW_ID_BITS` and `PRE_SPLIT_REGIONS` have been set when a table is created, the specified number of Regions are evenly split after the table creation.
```

**Review by shichun-0415 — APPROVED** [review]

**Pull request merged**